app/src/main/java/com/spotlight/spotlightapp/task/views/CurrentTaskFragment.kt

```diff
@@ -0,0 +1,212 @@
+    }
+
+    @Composable
+    private fun CurrentTaskLayout() {
+        val currentTask = currentTaskViewModel.currentTaskUIState.observeAsState()
+
+        currentTask.value?.apply {
+            Column(modifier = Modifier.padding(20.dp), verticalArrangement = Arrangement.Center) {
+                CurrentTaskView(task = task)
+                Spacer(modifier = Modifier.height(56.dp))
+                CurrentTaskButtons(willShowEditButtons, task.isFinished)
+            }
+        }
+    }
+
+    @Composable
+    private fun CurrentTaskView(task: Task) {
+        Column(modifier = Modifier.fillMaxWidth()) {
+            Text(
+                text = task.title, color = colorResource(id = R.color.primaryBlack),
+                fontSize = 20.sp, fontFamily = ComposeTextConfiguration.fontFamily,
+                fontWeight = FontWeight.SemiBold,
+                lineHeight = 24.sp)
+            Spacer(modifier = Modifier.height(12.dp))
+            Text(
+                text = task.description, color = colorResource(id = R.color.primaryBlack),
+                fontSize = 16.sp, fontFamily = ComposeTextConfiguration.fontFamily,
+                fontWeight = FontWeight.Normal,
+                lineHeight = 20.sp)
+        }
+    }
+
+    @Composable
+    private fun CurrentTaskButtons(willShowEditButtons: Boolean, isTaskFinished: Boolean) {
+        Row(
+            modifier = Modifier
+                .height(IntrinsicSize.Min)
+                .fillMaxWidth(),
+            verticalAlignment = Alignment.CenterVertically) {
+            when {
+                willShowEditButtons -> {
+                    CustomComposeViews.Button(
+                        modifier = Modifier
+                            .padding(end = 4.dp)
+                            .fillMaxHeight(),
+                        imageRes = R.drawable.ic_edit, labelText = null,
+                        buttonColors = ButtonDefaults.buttonColors(
+                            backgroundColor = colorResource(id = R.color.functionBlue),
+                            contentColor = colorResource(id = R.color.primaryWhite))) {}
+                    CustomComposeViews.Button(
+                        modifier = Modifier
+                            .padding(horizontal = 2.dp)
+                            .fillMaxHeight(),
+                        imageRes = R.drawable.ic_delete, labelText = null,
+                        buttonColors = ButtonDefaults.buttonColors(
+                            backgroundColor = colorResource(id = R.color.functionRed),
+                            contentColor = colorResource(id = R.color.primaryWhite))) {}
+                }
+                !isTaskFinished -> {
+                    CustomComposeViews.Button(
+                        modifier = Modifier
+                            .weight(1f)
+                            .padding(end = 4.dp)
+                            .fillMaxWidth(),
+                        imageRes = 0, labelText = "1 hr",
```

Comment:
Add a TODO comment regarding changing the label text for the time here to be dynamic.